apps/taskmasters/flowlord/handler.go

```diff
@@ -0,0 +1,313 @@
+package main
+
+import (
+	"encoding/json"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/go-chi/chi/v5"
+	gtools "github.com/jbsmith7741/go-tools"
+	"github.com/jbsmith7741/go-tools/appenderr"
+	"github.com/pcelvng/task"
+
+	tools "github.com/pcelvng/task-tools"
+	"github.com/pcelvng/task-tools/tmpl"
+	"github.com/pcelvng/task-tools/workflow"
+)
+
+func (tm *taskMaster) StartHandler() {
+	router := chi.NewRouter()
+	router.Get("/", tm.Info)
+	router.Get("/info", tm.Info)
+	router.Get("/refresh", tm.refreshHandler)
+	router.Get("/backload", tm.Backloader)
```

Comment:
I think it makes more sense to have a POST request here. Not a get request. Normal standard is that get requests do not have a request body.